config/resources/lpdc.json

```diff
@@ -0,0 +1,653 @@
+      },
+      "relationships": {
+        "identifiers": {
+          "target": "identifiers",
+          "predicate": "dct:identifier",
+          "cardinality": "one"
+        },
+        "related": {
+          "target":  "conceptual-public-services",
+          "predicate": "dct:relation",
+          "cardinality": "many"
+        },
+        "instances": {
+          "target": "public-services",
+          "predicate": "dct:source",
+          "cardinality": "many"
+        },
+        "competent-authority": {
+          "target": "public-organizations",
+          "predicate": "m8g:hasCompetentAuthority",
+          "cardinality": "many"
+        },
+        "participations": {
+          "target": "participations",
+          "predicate": "m8g:hasParticipation",
+          "cardinality": "many"
+        },
+        "requirements": {
+          "target": "requirements",
+          "predicate": "ps:hasRequirement",
+          "cardinality": "many"
+        },
+        "follows": {
+          "target": "rules",
+          "predicate": "cspv:follows",
+          "cardinality": "many"
+        },
+        "produces": {
+          "target": "financial-advantages",
+          "predicate": "cspv:produces",
+          "cardinality": "many"
+        },
+        "legal-resources": {
+          "target": "legal-resources",
+          "predicate": "m8g:hasLegalResource",
+          "cardinality": "many"
+        },
+        "contact-points": {
+          "target": "contact-points",
+          "predicate": "m8g:hasContactPoint",
+          "cardinality": "many"
+        },
+        "attachments": {
+          "target": "documents",
+          "predicate": "lpdcExt:attachment",
+          "cardinality": "many"
+        },
+        "lifecycle-status": {
+          "target": "lifecycle-status",
+          "predicate": "ps:lifecycleStatus",
+          "cardinality": "one"
+        },
+        "sectors": {
+          "target": "sectors",
+          "predicate": "m8g:sector",
+          "cardinality": "many"
+        },
+        "spatial": {
+          "target": "locations",
+          "predicate": "dct:spatial",
+          "cardinality": "many"
+        },
+        "target-audiences": {
+          "target": "target-audiences",
+          "predicate": "lpdcExt:targetAudience",
+          "cardinality": "many"
+        },
+        "thematic-areas": {
+          "target": "thematic-areas",
+          "predicate": "m8g:thematicArea",
+          "cardinality": "many"
+        },
+        "type": {
+          "target": "serviceTypes",
+          "predicate": "dct:type",
+          "cardinality": "many"
+        },
+        "costs": {
+          "target": "costs",
+          "predicate": "m8g:hasCost",
+          "cardinality": "many"
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/conceptual-public-service/"
+    },
+    "identifiers": {
+      "class": "adms:Identifier",
```

Comment:
It seems that this is defined as `adms:identifier` in the migration files:

```
<http://data.lblod.info/id/lpdc-ipdc/00411460f7c92d2124a67ea0f4cb5f85> a <http://www.w3.org/ns/adms#identifier> ;
    skos:notation 363 .
```

I think that needs to be capitalized to match the config here?

Reply:
Wait, that's the "structured identifier part" nevermind 😄.

Reply:
It's correct that the class is with a capital and this was wrong in the data. Well spotted! I've pushed a migration to change this.
Currently we only have one (made up) conceptual service about eID. All other services are "concrete".
It is indeed used for the PID of the services.